cyprecice/cyprecice.pyx

```diff
@@ -1211,6 +1211,223 @@ cdef class Interface:
         self.thisptr.readScalarData (data_id, vertex_id, _value)
         return _value
 
+    def write_block_vector_gradient_data (self, data_id, vertex_ids, gradientValues):
+        """
+        Writes vector gradient data given as block. This function writes values of specified vertices to a dataID.
+        Values are provided as a block of continuous memory. Values are stored in a numpy array [N x D] where N = number
```

Comment:
I'm not sure if the memory comment here is valuable for python users. I guess you copied this over from the c++ docs?

Reply:
It is from python bindings, https://github.com/kursatyurt/python-bindings/blob/bd7d0315c7db73b996a7879944c05ef645ca5a5c/cyprecice/cyprecice.pyx#L872-L875